src/Reflection/Adapter/ReflectionEnumBackedCase.php

```diff
@@ -14,6 +14,7 @@
 use function array_map;
 use function sprintf;
 
+/** @psalm-suppress MissingImmutableAnnotation */
```

Comment:
Can we attempt hunting for immutability? 🤔

Reply:
It's probably complicated, we already have https://github.com/Roave/BetterReflection/blob/6.5.x/src/Reflection/Adapter/ReflectionParameter.php#L26 and https://github.com/Roave/BetterReflection/blob/6.5.x/src/Reflection/Adapter/ReflectionNamedType.php#L12

Reply:
OK, let's roll with this as-is then, thanks :)